apps/labs/posts/implement-lapack-routines-in-stdlib.md

```diff
@@ -0,0 +1,210 @@
+    </td>
+    <td style="vertical-align: top; width: auto;">
+      <!-- Text column -->
+        <p>
+            One day, I reviewed all the available LAPACK routines from netlib-lapack, categorizing them based on difficulty and dependencies. I compiled this information into a list, which can be found at lapack-tracker-issue. My original approach was to implement the routines in a depth-first manner for each package, which led to the creation of several dependency trees, prioritizing easier implementations.
+        </p>
+        <p>
+            I quickly realized that the depth-first approach would not be feasible, as we did not have the luxury of years to develop and integrate the packages. Instead, I had a strict timeline of just three months to get up to speed, minimize code errors, automate certain processes, and still maintain a steady and positive pace in implementing the packages.
+        </p>
+        <p>
+            After a discussion with Athan, we decided on a two-pronged strategy to avoid potential bottlenecks: (1) continue working in a depth-first approach to maintain progress while PRs are under review, and (2) focus on implementing packages that are leaf nodes in most dependency trees, thereby establishing a solid foundation for future development OR simply *pickup low hanging fruits* :)
+        </p>
+    </td>
+  </tr>
+</table>
+
+## Challenges during Fortran to JS conversion
+
+1. Supporting both `row-major` and `column-major`.
+
+
+<table>
+  <tr>
+    <td style="vertical-align: top; width: auto;">
+        <p>
+            Fortran stores array elements in a column-major format, unlike C or JavaScript, which prefer row-major storage. Following the approach used in LAPACKE, we decided to introduce a new parameter, order, in each implementation to specify the storage layout. Based on the value of order, there would be distinct implementations and optimizations for each layout.
+        </p>
+        <p>
+            The order we loop through multidimensional arrays can have a big impact on speed. Fortran is as said column-major, Meaning consecutive elements of a column are stored next to each other in memory, and we should loop through arrays in this order order of columns unlike conventional looping over rows.
+        </p>
+    </td>
+    <td style="vertical-align: top; width: auto;">
+      <img src="/posts/implement-lapack-routines-in-stdlib/image-3.png" alt="alt text" style="width:3200px;height:300px;">
+    </td>
+  </tr>
+</table>
+
+Let's illustrate this with an example. Consider a 2D array `A` of size `5x4`. To iterate over the array in row-major order, we would loop through the rows first, followed by the columns. In contrast, to iterate over the array in column-major order, we would loop through the columns first, followed by the rows. The code snippet below demonstrates this concept.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+var A; var B; var i; var j;
```

Comment:
We should go ahead and use ES2015+.

Reply:
Instead I used the `dlacpy` function you provided.